lib/routes/4khd/article.ts

```diff
@@ -5,7 +5,7 @@ import { WPPost } from './types';
 const processImages = ($) => {
     $('a').each((_, elem) => {
         const $elem = $(elem);
-        const largePhotoUrl = $elem.attr('href').replace('i0.wp.com/pic', 'img');
+        const largePhotoUrl = $elem.attr('href')?.replace('pic.4khd.com', 'yt4.googleusercontent.com').replace('AsHYQ', 'AsYHQ').replace('l/AAA', 'I/AAA');
```

Comment:
Please remove `i0.wp.com` from the URL as well as it will interfere with hotlink template